.github/workflows/windows-msys2.yml

```diff
@@ -78,93 +81,89 @@ jobs:
           - { target: release, sys: ucrt64,  env: ucrt-x86_64  }
           - { target: release, sys: clang64, env: clang-x86_64 }
           - { target: release, sys: mingw32, env: i686         }
-    runs-on: windows-latest
-    needs: prepare
-    timeout-minutes: 45
 
     steps:
 
-      - name: Build environment setup
+      - name: Get CI dist tarball
+        uses: actions/download-artifact@v4
+        with:
+          name: gnucobol-ci source distribution msys2
+
+      - name: Extract CI dist tarball
+        shell: bash
+        run: tar -xvf gnucobol*.tar.* --strip-components=1
+
+      - name: Setup build environment
+        shell: pwsh
         run: |
-          if ("${{ matrix.target }}" -eq "release") {
+          If ("${{ matrix.target }}" -eq "release") {
             echo DISTDIR=GnuCOBOL_mingw >> $env:GITHUB_ENV
             echo CFGOPT="--with-pkgversion=GnuCOBOL-CI-MSYS2" >> $env:GITHUB_ENV
-          } else {
+          } Else {
             echo DISTDIR=GnuCOBOL_mingw_dbg >> $env:GITHUB_ENV
-            echo CFGOPT="--with-pkgversion=GnuCOBOL-CI-MSYS2-debug --enable-debug --enable-cobc-internal-checks --enable-hardening" >> $env:GITHUB_ENV
+            echo CFGOPT="--with-pkgversion=GnuCOBOL-CI-MSYS2-Debug --enable-debug --enable-cobc-internal-checks --enable-hardening" >> $env:GITHUB_ENV
           }
           # Common options (dependencies)
           echo CFGOPT="$env:CFGOPT --with-math=gmp --with-curses=ncursesw --with-xml2 --with-json=json-c" >> $env:GITHUB_ENV
           # Conditional inclusion of --with-db (MSYS2 does not provide 32bit builds for BDB any more)
-          if ("${{ matrix.sys }}" -ne "mingw32") {
+          If ("${{ matrix.sys }}" -ne "mingw32") {
             echo CFGOPT="$env:CFGOPT --with-db" >> $env:GITHUB_ENV
-          } else {
+          } Else {
```

Comment:
Just out of interest... What was the reason to capitalize if/else?

Reply:
Oh it is just to match what we have in the GC4 CI. I could as well do the opposite and change the GC4 to lowercase. I don't know what's the usual convention for PowerShell (have seen both used online).